docs/_data/options.ts

```diff
@@ -45,103 +45,127 @@ export const options: OptionsListItem[] = [
 		text: "",
 		textField: true,
 	},
-	{ name: " --apache", id: "apache", param: "--apache", desc: " Install Apache.", selected: true },
+	{
+		name: " --apache",
+		id: "apache",
+		param: "--apache",
+		desc: "Install Apache for htaccess support",
+		selected: true,
+	},
 	{ name: " --phpfpm", id: "phpfpm", param: "--phpfpm", desc: "Install PHP-FPM.", selected: true },
 	{
 		name: " --multiphp",
 		id: "multiphp",
 		param: "--multiphp",
-		desc: " Install Multi-PHP.",
+		desc: "Install Multiple PHP versions (5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0, 8.1, 8.2)",
 		selected: true,
 	},
 	{
 		name: " --vsftpd",
 		id: "vsftpd",
 		param: "--vsftpd",
-		desc: "Install Vsftpd.",
+		desc: "Use vsftpd for FTP: lightweight, minimalist, secure",
 		selected: true,
 		conflicts: "proftpd",
 	},
 	{
 		name: " --proftpd",
 		id: "proftpd",
 		param: "--proftpd",
-		desc: "Install ProFTPD.",
+		desc: "Use ProFTPD for FTP: advanced, more modular, supports LDAP",
 		selected: false,
 		conflicts: "vsftpd",
 	},
-	{ name: " --named", id: "named", param: "--named", desc: "Install Bind.", selected: true },
+	{
+		name: " --named",
+		id: "named",
+		param: "--named",
+		desc: "Manage your DNS in Hestia with custom nameservers",
+		selected: true,
+	},
 	{
 		name: " --mysql",
 		id: "mysql",
```

Comment:
Should this be `mariadb`?

Reply:
https://github.com/hestiacp/hestiacp/blob/9c1c973593b964aeb191620cc6aff5e01c79de3f/install/hst-install-ubuntu.sh#L63

It was always --mysql as Debian / Ubuntu decided to replace MariaDB with Mysql. 

I have allready added --mariadb to it. It should be fine to change it.